### PR TITLE
[11.0][ADD] sale_margin_sync: Recompute sale margin when related stock…

### PR DESCRIPTION
**Pull request by carlosdauden**
… move cost price is changed

@Tecnativa  TT19351

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
You can rebase for getting proper Travis after I fix it.

**Comment by carlosdauden** [comment]
Rebased

File: sale_margin_sync/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    'category': 'Stock',
+    'website': 'https://github.com/OCA/stock-logistics-workflow',
+    'author': 'Tecnativa, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'depends': [
+        'sale_margin',
+        'sale_stock',
+    ],
+    'data': [
```
Comment (sergio-teruel) [inline sale_margin_sync/__manifest__.py]: Remove empty key

File: sale_margin_sync/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Sale margin sync',
+    'summary': 'Recompute sale margin when stock move cost price is changed',
+    'version': '11.0.1.0.0',
+    'category': 'Stock',
+    'website': 'https://github.com/OCA/stock-logistics-workflow',
+    'author': 'Tecnativa, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
```
Comment (sergio-teruel) [inline sale_margin_sync/__manifest__.py:11]: Add auto_install

File: sale_margin_sync/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Sale margin sync',
+    'summary': 'Recompute sale margin when stock move cost price is changed',
+    'version': '11.0.1.0.0',
+    'category': 'Stock',
+    'website': 'https://github.com/OCA/stock-logistics-workflow',
```
Comment (sergio-teruel) [inline sale_margin_sync/__manifest__.py]: change website and category

**Review by sergio-teruel — COMMENTED** [review]
Minimal changes

**Comment by carlosdauden** [comment]
Changes done

**Review by sergio-teruel — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge

Let's go for the definitive one!

Please migrate it to v12 directly.

**Comment by carlosdauden** [comment]
Travis fixed

**Pull request merged**